tilequeue/command.py

```diff
@@ -577,7 +577,8 @@ def tilequeue_process(cfg, peripherals):
     assert n_simultaneous_query_sets > 0
     # reduce queue size when we're rendering metatiles to try and avoid the
     # geometry waiting to be processed from taking up all the RAM!
-    default_queue_buffer_size = max(1, 128 >> (2 * (cfg.metatile_size or 0)))
+    default_queue_buffer_size = max(
+        1, 128 >> ((2 * (cfg.metatile_size or 0))) + 1)
```

Comment:
I'm not sure what the relationship should be. It should probably be related to metatile size, but imho if we wanted to provide a one size fits all formula we might need to take the available memory into consideration. Alternatively, we can just move this to configuration and have the existing calculation be the default. Or maybe not have any code changes at all here, because the downstream queue sizes use this just as a default, and already support being configured individually.